SimpleBrowser/Network/WebRequestWrapper.cs

```diff
@@ -226,7 +226,19 @@ public string Host
                 this.webRequest.Host = value;
             }
         }
+        public System.Security.Cryptography.X509Certificates.X509CertificateCollection ClientCertificates

```

Comment:
Add System.Security.Cryptography.X509Certificates to the usings at the top of the file. Remove the namespace from here.